enterprise/pulsar.py

```diff
@@ -367,7 +367,7 @@ def __init__(self, toas, model, sort=True, drop_pintpsr=True, planets=True):
             "AstrometryEquatorial" if "AstrometryEquatorial" in model.components else "AstrometryEcliptic"
         )
 
-        self._pos_t = model.components[which_astrometry].ssb_to_psb_xyz_ICRS(model.get_barycentric_toas(toas)).value
+        self._pos_t = model.components[which_astrometry].ssb_to_psb_xyz_ICRS(model.get_barycentric_toas(toas).value)
```

Comment:
Add at the top:

`from astropy.time import Time`

Then change this line to:

`self._pos_t = model.components[which_astrometry].ssb_to_psb_xyz_ICRS(Time(model.get_barycentric_toas(toas),format='mjd')).value`

Reply:
Seems good, will do this afternoon! Sorry for the delay

Reply:
@vhaasteren I thought my second commit was failing because I didn't run black, but it looks like that induced way more changes. I didn't see anything official about this in the contributing guidelines. What are best practices here?

Reply:
Definitely need black!  But can't be the latest version, I think.... Check the dev requirements?  I'm on my phone and will look after I respond here...

Reply:
Yeah.  v23.

Reply:
Thanks so much Scott! I actually ran test/lint/docs this time (with correct black ver) so hopefully it'll be ok now :)

Reply:
Yeah, some of the old committed code was from before black. So if you run black on the whole file yourself, you re-format more than just what you changed. Not very neat. But it' not wrong to put it in, let's just commit it.